### PR TITLE
[12.0] [MIG] Mail Queue Job

### PR DESCRIPTION
**Pull request by ThomasBinsfeld**
Migration of mail_queue_job from 11.0 to 12.0.

### CODE REVIEW

File: mail_queue_job/__manifest__.py [hunk 1]
```diff
@@ -1,10 +1,10 @@
-# Copyright 2018 ACSONE SA/NV
+# Copyright 2019 ACSONE SA/NV
```
Comment (sbidoul) [inline mail_queue_job/__manifest__.py]: @ThomasBinsfeld 2018-2019

**Review by guewen — APPROVED** [review]

**Comment by sbidoul** [comment]
We have detected transaction deadlocks when there are email notifications on queue jobs. Marking this WIP until we have diagnosed better.

**Comment by lmignon** [comment]
@ThomasBinsfeld Any news on this one?

**Comment by ThomasBinsfeld** [comment]
@lmignon Not for now i'm afraid.

**Pull request closed without merging**